src/core/heating_cooling_loads.jl

```diff
@@ -286,11 +280,11 @@ struct CoolingLoad
         # determine the timeseries of loads_kw_thermal
         loads_kw_thermal = nothing
         loads_kw = nothing
-        if length(thermal_loads_ton) > 0
-            if !(length(thermal_loads_ton) / time_steps_per_hour ≈ 8760)
+        if length(thermal_loads_tonhours_per_time_step) > 0
+            if !(length(thermal_loads_tonhours_per_time_step) / time_steps_per_hour ≈ 8760)
                 error("Provided cooling load does not match the time_steps_per_hour.")
             end
-            loads_kw_thermal = thermal_loads_ton .* KWH_THERMAL_PER_TONHOUR
+            loads_kw_thermal = thermal_loads_tonhours_per_time_step .* (KWH_THERMAL_PER_TONHOUR * time_steps_per_hour)
```

Comment:
@Bill-Becker could you review the addition of `* time_steps_per_hour` here?

Reply:
Similar to the heating `fuel_loads...`, we don't want the `* time_steps_per_hour`.